crates/hdk/src/link.rs

```diff
@@ -33,16 +32,15 @@ pub use hdi::link::*;
 /// See [ `get_details` ] and get for more information about CRUD
 /// See [ `get_links` ] and [ `get_link_details` ] for more information about filtering by tag
 ///
-/// Generally links and CRUDs _do not interact_ beyond the fact that links need entry hashes to
+/// Generally links and CRUDs _do not interact_ beyond the fact that links need hashes to
```

Comment:
I don't understand what "interact" means. CRUDs being create and update actions, say I have a link to an entry. That entry has been created successfully. The entry is the referenced data.
What does interact mean here?

Reply:
@jost-s they don't interact, i don't have a specific definition of something they don't do

Reply:
Does it mean that the link to a hash is not affected by any CRUD operation that could be applied to the link target?

Reply:
@jost-s yeah that's part of not interacting, links simply don't care about CRUD and CRUD doesn't care about links

Reply:
Reading this paragraph again makes it clear for me.